components/TextControls/tests/unit/Theming/MDCFilledTextFieldThemingTests.m

```diff
@@ -0,0 +1,324 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialContainerScheme.h"
+#import "MaterialTextControls+Theming.h"
+#import "MaterialTextControls.h"
+
+@interface MDCFilledTextFieldThemingTest : XCTestCase
+@property(nonatomic, strong) MDCFilledTextField *textField;
+@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
+@property(nonatomic, strong) MDCTypographyScheme *typographyScheme;
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
+@end
+
+@implementation MDCFilledTextFieldThemingTest
+
+- (void)setUp {
+  [super setUp];
+  self.textField = [[MDCFilledTextField alloc] init];
+  self.colorScheme =
+      [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+  self.typographyScheme =
+      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
```

Comment:
Should we default these values to the latest enum values?

Reply:
Will change in a follow up.